pkgs/by-name/ms/mslicer/package.nix

```
@@ -0,0 +1,56 @@
+{
+  fetchFromGitHub,
+  lib,
+  libglvnd,
+  libxkbcommon,
+  nix-update-script,
+  rustPlatform,
+  vulkan-loader,
+  wayland,
+}:
+
+rustPlatform.buildRustPackage rec {
```

finalAttrs is generally preferred (makes it easier to override)
```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
```

---

fixed!